app/src/main/java/org/mozilla/focus/ScreenController.kt

```diff
@@ -117,7 +117,7 @@ object ScreenController {
             // for visibility in addition to existence.
             browserFragment.loadUrl(url)
         } else {
-            SessionManager.getInstance().createSession(Source.USER_ENTERED, url)
+            SessionManager.getInstance().createSession(source, url)
```

Comment:
...what other sources are there?

Reply:
VIEW, SHARE, from intents: it's in the next commit. I don't know what these are used for or if they're important, however...